pandas/core/ops.py

```diff
@@ -1660,18 +1660,19 @@ def _construct_result(left, result, index, name, dtype=None):
     not be enough; we still need to override the name attribute.
     """
     out = left._constructor(result, index=index, dtype=dtype)
-
+    out.__finalize__(left)
```

Comment:
shouldn't this be ``out = out.__finalize__(left)``?

Reply:
Fixed. Thanks!